### ISSUE TITLE
No Laravel 8 support

### ISSUE THREAD

**Issue by maki10** [UniSharp/laravel-filemanager#993]
Currently, this package not support the Laravel 8. For the list bellow expect a new parameter to be `|| ^8.0`

```
"illuminate/config": "5.4.* || 5.5.* || 5.6.* || 5.7.* || 5.8.* || ^6.0 || ^7.0",
"illuminate/container": "5.4.* || 5.5.* || 5.6.* || 5.7.* || 5.8.* || ^6.0 || ^7.0",
"illuminate/filesystem": "5.4.* || 5.5.* || 5.6.* || 5.7.* || 5.8.* || ^6.0 || ^7.0",
"illuminate/http": "5.4.* || 5.5.* || 5.6.* || 5.7.* || 5.8.* || ^6.0 || ^7.0",
"illuminate/support": "5.4.* || 5.5.* || 5.6.* || 5.7.* || 5.8.* || ^6.0 || ^7.0",
```

When we can expect support for the Laravel 8?

**Comment by Itsmaqsud** [comment]
I'm waiting too

**Comment by shankhadevpadam** [comment]
Unable to install in laravel 8. 

```
Problem 1
    - Root composer.json requires unisharp/laravel-filemanager ^2.1 -> satisfiable by unisharp/laravel-filemanager[v2.1.0].
    - unisharp/laravel-filemanager v2.1.0 requires illuminate/config 5.4.* || 5.5.* || 5.6.* || 5.7.* || 5.8.* || ^6.0 || ^7.0 -> found illuminate/config[v5.4.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev] but it conflicts with another require.

You are using a snapshot build of Composer 2, which may be the cause of the problem. Run `composer self-update --stable` and then try again. In case it solves the problem, please report an issue mentioning Composer 2.

```

**Comment by imbenuk** [comment]
any idea when there will be laravel 8 support

**Comment by denniev** [comment]
I installed the master version and it dit the trick. In composer.json:

 "unisharp/laravel-filemanager": "master",

tnx for a good package!

**Comment by MuhammadIhsanAnsory** [comment]
> I installed the master version and it did the trick. In composer.json:
> 
> "unisharp/laravel-filemanager": "master",
> 
> tnx for a good package!

goog idea ! Its work :D

**Comment by NgocMinh96** [comment]
> I installed the master version and it did the trick. In composer.json:
> 
> "unisharp/laravel-filemanager": "master",
> 
> tnx for a good package!

Thanks guy! 
"master" is dev version?

**Comment by ziyakh542** [comment]
try this package: https://github.com/mafftor/laravel-file-manager
this package is work with laravel v8 
It has better interfaces and some extra features
But you must activate the private folder feature from the file located in the following path: config/lfm.php 
line 48     'allow_private_folder' => true, 

  /*
    |--------------------------------------------------------------------------
    | Folder privacy
    |--------------------------------------------------------------------------
    |
    | If both options are set to false, the shared folder will be activated.
    |
    */

    'allow_private_folder' => true,

**Comment by streamtw** [comment]
Fixed in `v2.2.0`, thanks for reporting this.
